CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ using MetricFu
 
 ## Unreleased
 
+* Prevent incompatible versions of metric_fu and flog to get installed together
```

Comment:
Well, as this is a change, I'd let it in there. I have no problem with opening an issue addressing it and omitting it from the change log.